### DISCUSSION TITLE
"Breaking, no choices" for everything I type

### DISCUSSION THREAD

**Discussion by M-Gonzalo** [community/community#12482] [Copilot Conversations] [answered]
There's been an issue with GH Copilot since a few minutes ago. I tried restarting the IDE and also deactivating the latest installed extensions just in case, but it's not working

```
[INFO] [default] [2022-03-07T00:15:36.199Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 324 ms
[INFO] [ghostText] [2022-03-07T00:15:36.210Z] Breaking, no choices
```

I'm using Visual Studio Code on Manjaro Linux KDE x86-64 up-to-date.
The service was working earlier today.

**Comment by davecheney** [comment]
Hello. I'm sorry you're having difficulties. I can't seem to reproduce the problem myself. Is it all code that is no longer generating completions, or only some?

**Reply by M-Gonzalo** [reply]
I'm actually receiving some completions, based on local suggestions.
It's weird, because sometimes it seems to work but most of the times not. Can it have something to do with the fact I'm working with Apollo/GraphQL? I only ask because today is the first time I code using these tools and it seems like a good enough coincidence

**Reply by davecheney** [reply]
I don't think copilot has much to offer for those file types.

**Reply by M-Gonzalo** [reply]
I'm sure that's true, only i discovered the problem trying to write a mock-up list if users as an array of objects in JavaScript, a case where Copilot should shine. It's 2:30 in the night here so I'll have to get back to this in a couple of hours though. Sorry

**Comment by M-Gonzalo** [comment, marked as answer]
@davecheney nevermind, it's working now...
Thanks for the quick reply though
